### PR TITLE
docs: add `sendRedirect` usage

### PR DESCRIPTION
**Pull request by yuyinws**
### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description
Add usage for `sendRedirect`

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by 06377903375** [comment]
<template>
  <h1>Hello world!</h1>
</template>

**Comment by 06377903375** [comment]
<template>
  <h1>Hello world!</h1>
</template>

**Review by TheAlexLichter — APPROVED** [review]

**Review by atinux — APPROVED** [review]

File: docs/2.guide/2.directory-structure/1.server.md [hunk 1]
```diff
@@ -255,6 +255,16 @@ export default defineEventHandler((event) => {
 })
 ```
 
+### Sending Redirect
+
+```ts [server/api/foo.get.ts]
+export default defineEventHandler(async (event) => {
+  return sendRedirect(event, '/path/redirect/to', 302)
+  // or
+  // await sendRedirect(event, '/path/redirect/to', 302)
```
Comment (pi0) [inline docs/2.guide/2.directory-structure/1.server.md:264]: awaiting only is not preferred. We should always return to ensure control flow terminates at this point.

Comment (pi0) [inline docs/2.guide/2.directory-structure/1.server.md:264]:
```suggestion
```

File: docs/2.guide/2.directory-structure/1.server.md [hunk 2]
```diff
@@ -255,6 +255,16 @@ export default defineEventHandler((event) => {
 })
 ```
 
+### Sending Redirect
+
+```ts [server/api/foo.get.ts]
+export default defineEventHandler(async (event) => {
+  return sendRedirect(event, '/path/redirect/to', 302)
+  // or
```
Comment (pi0) [inline docs/2.guide/2.directory-structure/1.server.md:263]:
```suggestion
```

File: docs/2.guide/2.directory-structure/1.server.md (hunk 1 above)
Comment (manniL) [reply docs/2.guide/2.directory-structure/1.server.md:264]: Good catch 👍🏻

**Review by danielroe — APPROVED** [review]

**Pull request merged**